sql_generators/glean_usage/templates/event_monitoring_live_v1.materialized_view.sql

```diff
@@ -1,10 +1,14 @@
 CREATE MATERIALIZED VIEW
 IF
   NOT EXISTS `{{ project_id }}.{{ derived_dataset }}.event_monitoring_live_v1`
+  PARTITION BY DATE(submission_date_timestamp)
```

Comment:
PARTITION BY only seems to work on TIMESTAMP. Possibly because the underlying live tables are also partitioned on a timestamp field.

Reply:
In BigQuery console it does provide this suggestion: `Partitioning column of the incremental materialized view must either match partitioning column or pseudo-column of the base table, or be a TIMESTAMP_TRUNC over it.`